Makefile

```
@@ -98,6 +98,9 @@ dist/subctl-%.tar.gz: cmd/bin/subctl-%
 	mkdir -p dist
 	tar -czf $@ --transform "s/^cmd.bin/subctl-$(VERSION)/" $<
 
+create_checksums: $(CROSS_TARBALLS)
```

Don’t use a phony target for a specific file; use the file as the target:

```suggestion
dist/subctl-checksums.txt: $(CROSS_TARBALLS)
```